INSTALL.md

```diff
@@ -80,14 +80,29 @@ You may also want to adjust the **scheduler: fork** to **scheduler: sge** if you
 Step 3: Database Setup
 ----------------------
 
-The OrthoMCL also requires a SQL database such as [MySQL](http://www.mysql.com/) to be setup in order to load and process some of the results.  Both an account and a separate database need to be created specifically for OrthoMCL.
+The OrthoMCL also requires a [MySQL](http://www.mysql.com/) database to be setup in order to load and process some of the results.  An account needs to be created specifically for OrthoMCL. A special OrthoMCL configuration file needs to be generated with parameters and database connection information.  This can be generated automatically with the script **scripts/setup_database.pl**. There are two options for running this script as outlined below:
 
-Once the database is setup, a special OrthoMCL configuration file needs to be generated with parameters and database connection information.  This can be generated automatically with the script **scripts/setup_database.pl** as follows:
+Option 1: 
+	If you have a previously created database you can run the script with the option: --no-create-database
+	$ perl scripts/orthomcl-setup-database.pl --user orthomcl --password orthomcl --host localhost --database orthomcl --outfile orthomcl.conf --no-create-database
```

Comment:
Can you check to make sure this displays correctly on github https://github.com/JenCabral/orthomcl-pipeline/blob/feature/setup-database/INSTALL.md.  You may need an extra newline here before the `$perl scripts/..`?